packages/hooks/src/useCheck10kLimit.ts

```diff
@@ -0,0 +1,15 @@
+import { useStore } from '@deriv/stores';
+import useMaxWithdrawAmount from './useMaxWithdrawAmount';
+
+const useCheck10kLimit = (): boolean | undefined => {
```

Comment:
@hamid-deriv Same here, maybe we can return `false` as the default value.

Reply:
Fixed.